server/charges.py

```diff
@@ -48,27 +48,6 @@ def amount_to_charge(opportunity):
     return quantize(total)
 
 
-def amount_to_charge_stripe(form=None):
-    """
-    Determine the amount to charge. This depends on whether the payer agreed
-    to pay fees or not. If they did then we add that to the amount charged.
-    Stripe charges 2.2% + $0.30 on single charges and 2.2% + 0.5% + $0.30
-    on recurring charges.
-
-    https://support.stripe.com/questions/can-i-charge-my-stripe-fees-to-my-customers
-    """
-    amount = float(form["amount"])
-    logging.info(f"pay_fees_value: {form['pay_fees_value']}")
-    if form["pay_fees_value"]:
-        if form["installment_period"] == None:
-            total = (amount + 0.30) / (1 - 0.022)
-        else:
-            total = (amount + 0.30) / (1 - 0.027)
-    else:
-        total = amount
-    return quantize(total)
-
-
 def quantize(amount):
```

Comment:
<!-- Suggestions propose improvements to the current subject. -->
**suggestion:** We might want to also move `quantize` and `TWOPLACES` to utils.py to avoid duplicate code and then import here.

Reply:
Yeah, great point. The reason why it's in two different places is because it's currently used by the nightly load, but I will remove that duplicate code once we're able to turn off the nightly load (sometime in early 2024, likely)